.github/workflows/Pkg EdFi.Ods.CodeGen.yml

```diff
@@ -81,7 +88,7 @@ jobs:
 
     - name: Upload EdFi.Ods.CodeGen NugetPackage
       if: success()
-      uses: actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8 # v4.3.0
+      uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
```

Comment:
Had to downgrade this action because for some reason v4.3.0 triggers the next error in the `Rebuild Database Templates` workflow:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/actions/runs/8287303811/job/22679305890
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/assets/158314644/ff8aa4cf-e314-4b69-9827-93a0475e1d87)